### PR TITLE
More Phase Guns

### PR DESCRIPTION
**Pull request by Anewbe**
Phase guns now come in three (technically four) flavors:
- Phase Pistols have the same number of shots, but each shot does 40 damage to mobs, rather than 60. Explorers no longer get these right off the bat, they've been replaced with the option below. These _may be_ added in elsewhere in the future.
- Phase Carbines are larger Phase Pistols, with 10 shots per cell, doing 50 damage to mobs. The total damage output has been increased over the current Phase Pistol, assuming you hold it in both hands to fire. It also has a longer range than the Phase Pistol. This is now the default Explorer weapon.
- Phase Rifles are a larger version of the Phase Carbine. It has 16 shots per cell, does 60 damage per hit, and has more or less the entire screen within its range. It requires both hands to be properly wielded, but is otherwise an upgrade. This will be order-able from Cargo, or made in Science.
- The Phase Cannon is a massive two-handed phase weapon, that can only be made in Science. It has 24 shots per _weapon cell_, does 75 total damage to animals, and the shots can cross an entire screen, but it definitely requires both hands to use.

TODO:

- [x] Put all of the weapons where they're supposed to go.

EDIT: Halved the contents of the hunting rifle crate to match essentially every other weapon crate. Also removed the lethal rounds, there're other crates to buy if you want lethal bullets, or just find a hacked autolathe.
EDIT2: Got mad, moved all of the RnD designs into new files in a new folder.

### CODE REVIEW

**Comment by Atermonera** [comment]
Doesn't compile

**Comment by Anewbe** [comment]
Yeah, missed a file, give Travis a minute to check again.

**Comment by Atermonera** [comment]
Research designs are bork

**Comment by Anewbe** [comment]
I'm damn certain that was already in there, but I'll look again...

File: code/modules/projectiles/guns/energy/phase.dm [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// Phase weapons go here
+
+/obj/item/weapon/gun/energy/phasegun
+	name = "phase carbine"
+	desc = "The NT EW26 Artemis is a downsided energy weapon, specifically designed for use against wildlife."
```
Comment (Atermonera) [inline code/modules/projectiles/guns/energy/phase.dm]: downsized?

File: code/modules/research/designs/circuits.dm [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+	req_tech = list(TECH_DATA = 1)
+	build_path = /obj/item/weapon/circuitboard/arcade/battle
+	sort_string = "MAAAA"
+
+/datum/design/circuit/oriontrail
+	name = "orion trail arcade machine"
+	id = "oriontrail"
+	req_tech = list(TECH_DATA = 1)
+	build_path = /obj/item/weapon/circuitboard/arcade/orion_trail
+	sort_string = "MAAAA"
```
Comment (Atermonera) [inline code/modules/research/designs/circuits.dm]: Duplicate

File: code/modules/research/designs/pdas.dm [hunk 3]
```diff
@@ -0,0 +1,92 @@
+/datum/design/item/pda
+	name = "PDA design"
+	desc = "Cheaper than whiny non-digital assistants."
+	id = "pda"
+	req_tech = list(TECH_ENGINEERING = 2, TECH_POWER = 3)
+	materials = list(DEFAULT_WALL_MATERIAL = 50, "glass" = 50)
+	build_path = /obj/item/device/pda
+	sort_string = "VAAAA"
```
Comment (Atermonera) [inline code/modules/research/designs/pdas.dm]: 90% sure this is a duplicate

File: code/modules/research/designs/uncommented.dm [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+/datum/design/rust_core_control
+	name = "Circuit Design (RUST core controller)"
+	desc = "Allows for the construction of circuit boards used to build a core control console for the RUST fusion engine."
+	id = "rust_core_control"
+	req_tech = list("programming" = 4, "engineering" = 4)
+	build_type = IMPRINTER
+	materials = list("glass" = 2000, "sacid" = 20)
+	build_path = "/obj/item/weapon/circuitboard/rust_core_control"
+
+datum/design/rust_fuel_control
```
Comment (Atermonera) [inline code/modules/research/designs/uncommented.dm]: `/datum...` for this and every datum hereafter in this file

File: code/modules/research/designs/uncommented.dm [hunk 5]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (Atermonera) [inline code/modules/research/designs/uncommented.dm]: Every datum in here is missing a sort strings. Not an issue unless they're uncommented, but the instructions are literally to just uncomment if the conditions are met, soooo...

**Comment by Atermonera** [comment]
> 2 DAAAA
> 2 KAAAA
> 2 KAAAB
> 2 MAAAA
> 2 VAAAA
> 2 XAAAA
That's the complete list of duplicate strings.
It corresponds to:
XAAAA: `/datum/design/aimodule/reset`, `/datum/design/circuit/aicore`
MAAAA: `/datum/design/circuit/arcademachine`, `/datum/design/circuit/oriontrail`
DAAAA: `/datum/design/circuit/seccamera`, `/datum/design/item/powercell/basic`
KAAAA: `/datum/design/circuit/ordercomp`, `/datum/design/item/weapon/mining/jackhammer`
VAAAA: `/datum/design/circuit/shield/bubble`, `/datum/design/item/pda`
KAAAB: `/datum/design/item/weapon/mining/drill`, `/datum/design/circuit/supplycomp`

**Pull request merged**